pypeit/core/flexure.py

```diff
@@ -459,24 +459,20 @@ def spec_flexure_slit(slits, slitord, slit_bpm, sky_file, method="boxcar", speco
                 # Calculate the shift
                 fdict = spec_flex_shift(obj_sky, sky_spectrum, sky_lines,
                                         mxshft=mxshft, excess_shft=excess_shft)
-                punt = False
+
                 if fdict is None:
                     msgs.warn("Flexure shift calculation failed for this spectrum.")
                     if sv_fdict is not None:
                         msgs.warn("Will used saved estimate from a previous slit/object")
                         fdict = copy.deepcopy(sv_fdict)
                     else:
-                        # One does not exist yet
-                        # Save it for later
-                        return_later_sobjs.append([islit, ss])
-                        punt = True
+                        # One does not exist yet -- break out of the FOR loop
+                        break
+                        # # Save it for later
+                        # return_later_sobjs.append([islit, ss])
```

Comment:
This basically skips the code at lines 481-499 and `return_later_sobjs` is never used. It's probably better to comment that part of the code and add a note. I would also check with the person that wrote this part of code (maybe @profxj) if it's OK with it.

Reply:
I believe this change was purely cosmetic (removing the `punt` business and putting the `break` where `punt = True` was set), but I have reverted it alongside the other changes as mentioned above.